<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTIN541610" failures="1" tests="6" name="testRunner.TestRunner_TestNG" time="239.940" errors="1" timestamp="2025-07-15T10:25:52 IST" skipped="0">
  <testcase classname="testRunner.TestRunner_TestNG" name="runScenario" time="41.306"/>
  <testcase classname="testRunner.TestRunner_TestNG" name="runScenario" time="20.509"/>
  <testcase classname="testRunner.TestRunner_TestNG" name="runScenario" time="21.769"/>
  <testcase classname="testRunner.TestRunner_TestNG" name="runScenario" time="41.474">
    <error message="timeout: Timed out receiving message from renderer: 29.989
  (Session info: chrome=137.0.7151.69)
Build info: version: &#039;4.34.0&#039;, revision: &#039;707dcb4246*&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;21.0.5&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [32b304a6596b30ded92f036ea95ab385, goBack {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\2403720\AppData\Lo...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:59918}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:59918/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 32b304a6596b30ded92f036ea95ab385" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: timeout: Timed out receiving message from renderer: 29.989
  (Session info: chrome=137.0.7151.69)
Build info: version: '4.34.0', revision: '707dcb4246*'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [32b304a6596b30ded92f036ea95ab385, goBack {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\2403720\AppData\Lo...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:59918}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:59918/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 32b304a6596b30ded92f036ea95ab385
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:544)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:617)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:621)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteNavigation.back(RemoteWebDriver.java:1066)
at stepsDefination.TestCase010.the_user_navigates_back(TestCase010.java:31)
at ✽.the user navigates back(file:///C:/Users/2403720/eclipse-workspace/CourseraAutomation/features/testCase010.feature:8)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.TestRunner_TestNG" name="runScenario" time="59.029">
    <failure message="expected [3190] but found [31901]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [3190] but found [31901]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepsDefination.TestCase006.the_application_should_display_the_purchase_summary(TestCase006.java:79)
at ✽.the application should display the purchase summary(file:///C:/Users/2403720/eclipse-workspace/CourseraAutomation/features/testCase006.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.TestRunner_TestNG" name="runScenario" time="55.853"/>
</testsuite> <!-- testRunner.TestRunner_TestNG -->
